### PR TITLE
Longw/lcm

### PR DESCRIPTION
**Pull request by wanlonghenry**
lcm cluster support for provisioned cluster

### CODE REVIEW

File: scripts/onboarding/templates/arc-k8s-extension-provision-msi-auth/existingClusterOnboarding.json [hunk 1]
```diff
@@ -0,0 +1,202 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
```
Comment (ganga1980) [inline scripts/onboarding/templates/arc-k8s-extension-provision-msi-auth/existingClusterOnboarding.json:2]: MSI auth template & parameter file needs to be updated accordingly with changes in Kaveesh merged and once his PR merged.

File: charts/azuremonitor-containers/templates/omsagent-arc-k8s-crd.yaml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-{{- if or ( contains "microsoft.kubernetes/connectedclusters" (.Values.Azure.Cluster.ResourceId | lower) ) ( contains "microsoft.kubernetes/connectedclusters" (.Values.omsagent.env.clusterId | lower)) }}
+{{- if or ( contains "microsoft.kubernetes/connectedclusters" (.Values.Azure.Cluster.ResourceId | lower)) ( contains "microsoft.kubernetes/connectedclusters" (.Values.omsagent.env.clusterId | lower)) ( contains "microsoft.hybridcontainerservice/provisionedclusters" (.Values.Azure.Cluster.ResourceId | lower)) ( contains "microsoft.hybridcontainerservice/provisionedclusters" (.Values.omsagent.env.clusterId | lower)) }}
```
Comment (ganga1980) [inline charts/azuremonitor-containers/templates/omsagent-arc-k8s-crd.yaml:1]: This if clause can be simplified with just checking ".Values.Azure.Cluster.ResourceId" and we can remove the conditions related to ".Values.omsagent.env.clusterId"  which is not required. This was added in the begining when we dont have extension model.

Comment (wanlonghenry) [reply charts/azuremonitor-containers/templates/omsagent-arc-k8s-crd.yaml]: changed to only check resourceId

**Comment by wanlonghenry** [comment]
change commits target

**Pull request closed without merging**